### ISSUE TITLE
Build fails on windows 10

### ISSUE THREAD

**Issue by n-shift** [tauri-apps/tauri#1433]
**Describe the bug**
After following tutorial and running `yarn tauri dev` i am getting webview build error.

**To Reproduce**
Run `yarn tauri dev` in project created with `yarn tauri init`

**Expected behavior**
Build succeed

**Platform and Versions:**

OS: Windows_NT(10.0.18363) - win32/x64
Node: 15.11.0
NPM: 7.6.0
Yarn: 1.22.4
Rustc: 1.51.0

**Additional context**

**Stack Trace**
```
error: failed to run custom build command for `webview-official-sys v0.1.2`

Caused by:
  process didn't exit successfully: `C:\Users\dotco\.code\embed\tune\src-tauri\target\debug\build\webview-official-sys-529333c4b086818f\build-script-build` (exit code: 1)
  --- stdout
  cargo:rustc-link-lib=windowsapp
  cargo:rustc-link-lib=user32
  cargo:rustc-link-lib=oleaut32
  cargo:rustc-link-lib=ole32
  cargo:rustc-link-search=C:\Users\dotco\.cargo\registry\src\github.com-1ecc6299db9ec823\webview-official-sys-0.1.2\webview-official/script/Microsoft.Web.WebView2.0.9.488/build/native\x64
  cargo:rustc-link-lib=WebView2Loader.dll
  cargo:rerun-if-changed=webview-official/webview.h
  cargo:rerun-if-changed=webview-official/webview.cc
  TARGET = Some("x86_64-pc-windows-msvc")
  OPT_LEVEL = Some("0")
  HOST = Some("x86_64-pc-windows-msvc")
  CXX_x86_64-pc-windows-msvc = None
  CXX_x86_64_pc_windows_msvc = None
  HOST_CXX = None
  CXX = None
  CXXFLAGS_x86_64-pc-windows-msvc = None
  CXXFLAGS_x86_64_pc_windows_msvc = None
  HOST_CXXFLAGS = None
  CXXFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  DEBUG = Some("true")
  CXX_x86_64-pc-windows-msvc = None
  CXX_x86_64_pc_windows_msvc = None
  HOST_CXX = None
  CXX = None
  CXXFLAGS_x86_64-pc-windows-msvc = None
  CXXFLAGS_x86_64_pc_windows_msvc = None
  HOST_CXXFLAGS = None
  CXXFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  CXX_x86_64-pc-windows-msvc = None
  CXX_x86_64_pc_windows_msvc = None
  HOST_CXX = None
  CXX = None
  CXXFLAGS_x86_64-pc-windows-msvc = None
  CXXFLAGS_x86_64_pc_windows_msvc = None
  HOST_CXXFLAGS = None
  CXXFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  CXX_x86_64-pc-windows-msvc = None
  CXX_x86_64_pc_windows_msvc = None
  HOST_CXX = None
  CXX = None
  CXXFLAGS_x86_64-pc-windows-msvc = None
  CXXFLAGS_x86_64_pc_windows_msvc = None
  HOST_CXXFLAGS = None
  CXXFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  running: "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\bin\\amd64\\cl.exe" "-nologo" "-MD" "-Z7" "-Brepro" "-I" "webview-official/webview.h" "-I" "webview-official/script" "-W4" "-FoC:\\Users\\dotco\\.code\\embed\\tune\\src-tauri\\target\\debug\\build\\webview-official-sys-33fd681158bc76e1\\out\\webview-official/webview.o" "-c" "webview-official/webview.cc"
  webview.cc
  C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\include\ppltasks.h(1523): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
  c:\users\dotco\.cargo\registry\src\github.com-1ecc6299db9ec823\webview-official-sys-0.1.2\webview-official\webview.h(195): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
  c:\users\dotco\.cargo\registry\src\github.com-1ecc6299db9ec823\webview-official-sys-0.1.2\webview-official\webview.h(805): fatal error C1083: Cannot open include file: 'winrt/Windows.Foundation.Collections.h': No such file or directory
  exit code: 2

  --- stderr


  error occurred: Command "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\bin\\amd64\\cl.exe" "-nologo" "-MD" "-Z7" "-Brepro" "-I" "webview-official/webview.h" "-I" "webview-official/script" "-W4" "-FoC:\\Users\\dotco\\.code\\embed\\tune\\src-tauri\\target\\debug\\build\\webview-official-sys-33fd681158bc76e1\\out\\webview-official/webview.o" "-c" "webview-official/webview.cc" with args "cl.exe" did not execute successfully (status code exit code: 2).


warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by nothingismagick** [comment]
Thankyou, but we are no longer accepting issues on alpha tauri.

https://github.com/tauri-apps/tauri/issues/1253

**Comment by tiesselune** [comment]
@shift-d Hi, I had the same problem with the same error message, and I struggled a bit before finding a solution. I updated MSBuild Tools for C++ by going in the updater, hit modify, and unchecked C++ build tools before re-checking it, which seems to have reset it (and uninstalled build tools for desktop .Net). Then waited for it to re-install. Something must have been missing. 

I tried launching `npm run tauri dev` again and this time it compiled in no time. Hope that helps!
